Dockerfile

```diff
@@ -27,19 +27,19 @@ RUN git clone --depth=1 https://github.com/layer5io/wrk2 && cd wrk2 && make
 FROM alpine:3.14 as seed_content
 RUN apk add --no-cache curl
 WORKDIR /
-# bundling filters
+# bundling filters 
 RUN curl -L -s `curl -s https://api.github.com/repos/layer5io/wasm-filters/releases/latest | grep "browser_download_url" | cut -d : -f 2,3 | tr -d '"'` -o wasm-filters.tar.gz \
     && mkdir -p /seed_content/filters/binaries \
     && tar xzf wasm-filters.tar.gz --directory=/seed_content/filters/binaries
 
-# bundling patterns
+# bundling patterns 
 RUN curl -L -s https://github.com/service-mesh-patterns/service-mesh-patterns/tarball/master -o service-mesh-patterns.tgz \

```

Comment:
I'm curious as to why these two approaches are different between filters and patterns. Is this because one set of artifacts are binaries and the other yamls?

Reply:
Patterns are directly available on github api. 
To get filters, we need to download the release in wasm-filters repo

Reply:
Let's make releases in the other repos, too, so that we are only ever pulling released versions of the content.